.evergreen.yml

```diff
@@ -291,6 +291,9 @@ functions:
       working_dir: spruce
       script: |
         export PATH=/opt/nodejs/node-v16.13.0-linux-x64/bin:$PATH
+        chmod +x appVersion.sh
+        . ./appVersion.sh
+        echo "Building Production version: $REACT_APP_VERSION"
```

Comment:
This loads the output of this script into an env variable to be used during the build stage.